### PR TITLE
Add SAN_PRICE_ABSOLUTE_CHANGE

### PR DESCRIPTION
**Pull request by kalkov**
* Add `=SAN_PRICE_ABSOLUTE_CHANGE`
* Add a dedicated test file which is using stubbing
* Extract common functions into `test/helper.js`
* Run mocha recursively

### CODE REVIEW

File: test/functions/priceAbsoluteChangeTest.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+require('../helper.js')
+
+describe('priceAbsoluteChange_', () => {
+  const slug = 'santiment'
+  const from = '2019-01-01T00:00:00.000Z'
+  const to = '2019-01-03T23:59:59.000Z'
```
Comment (IvanIvanoff) [inline test/functions/priceAbsoluteChangeTest.js]: If we use only `Date`(without time) what will be the time set for the `from` and `to` params? Is it always 00:00:00, or the `to` will be set to `23:59:59`? If it's not the second case - then the test does not represent a real scenario

Comment (kalkov) [reply test/functions/priceAbsoluteChangeTest.js]: Good catch, fixed. 
```js
  const from = '2019-01-01T00:00:00.000Z'
  const to = '2019-01-03T00:00:00.000Z'
```

Comment (IvanIvanoff) [reply test/functions/priceAbsoluteChangeTest.js]: In such case I would rather expect that the `from` and `to` are both inclusive, but in this case `from` is inclusive and `to` is exclusive

Comment (kalkov) [reply test/functions/priceAbsoluteChangeTest.js]: This is the current behavior of the API. I have executed a graphQL query, copied and modified the response into the test.

**Pull request merged**